packages/nextra-theme-docs/src/components/select.tsx

```diff
@@ -54,13 +60,14 @@ export function Select({
                       active
                         ? 'bg-primary-50 text-primary-500 dark:bg-primary-500/10'
                         : 'text-gray-800 dark:text-gray-100',
-                      'relative cursor-pointer select-none whitespace-nowrap py-1.5 pl-3 pr-9'
+                      'relative cursor-pointer select-none whitespace-nowrap py-1.5',
+                      'ltr:pl-3 ltr:pr-9 rtl:pr-3 rtl:pl-9'
                     )
                   }
                 >
                   {option.name}
                   {option.key === selected.key ? (
-                    <span className="absolute inset-y-0 right-0 flex items-center pr-3">
+                    <span className="absolute inset-y-0 ltr:right-3 rtl:left-3 flex items-center">
```

Comment:
fixes (check icon should be on the left side)
<img width="241" alt="image" src="https://user-images.githubusercontent.com/7361780/184452963-959fda5e-8118-4437-bc5a-cd1fd7381bfc.png">